### DISCUSSION TITLE
simple:i want snow falling on vscode screen

### DISCUSSION THREAD

**Discussion by RIPISDRAD** [microsoft/vscode-discussions#1705] [Extension Development QnA] [answered]
hi guys
as the title said
i prefer to do it out of extension but people told me u cant.
then extension it is.
what i want at top is, to make a html file and add it to the vscode ,codes . so it reads and do it as it was it's developing plan to do
and the html file has snow animations ,so now it snows.
in other words i want permanent change in developer tools vscode
in other words u can make a site snows now imagine snows stays and vscode comes instead of the site , so how do i do this adding html to vscode
if it is not possible to add html ,to the base code via vscode or its extensions . what can I do to add an snow animation like to it with an extension so i have sth pretty thanks for helping

**Comment by robconery** [comment, marked as answer]
This might be a good question to ask on StackOverflow as it's more conceptual than something specific the team can help with.
